.github/workflows/w3c-integration-test.yml

```diff
@@ -0,0 +1,21 @@
+name: Run w3c tests on push
+
+on: [push]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2
+
+      - name: Install and Build 🔧
+        run: |
+          npm install --ignore-scripts
+          #npx lerna bootstrap --scope={propagation-tests,@opentelemetry-api} --include-dependencies
+          #npx lerna link ----force-local
```

Comment:
Anyone know how to use lerna really well? This works locally, but when I try it here in the ubuntu machine, I get ```PluginLoader#load: could not load plugin @opentelemetry/plugin-http of module http. Error: Cannot find module '@opentelemetry/plugin-http'``` in the next step.

Reply:
I'm currently just running `npm i` in the bash script to get around this.

Reply:
It's because your bootstrap on the line above excludes the http plugin from the bootstrap.